src/components/plots/styling/violin/ViolinControls.jsx

```diff
@@ -59,9 +59,14 @@ const ViolinControls = (props) => {
 };
 
 ViolinControls.propTypes = {
-  config: PropTypes.object.isRequired,
+  config: PropTypes.object,
   onUpdate: PropTypes.func.isRequired,
   setSearchedGene: PropTypes.func.isRequired,
   cellSets: PropTypes.object.isRequired,
 };
+
+ViolinControls.defaultProps = {
+  config: null,
```

Comment:
What is the use of the controls if config can be null? Why dont we enforce it like before?